### PR TITLE
NATIVE-497: Switch from git submodules to subtrees

### PR DESCRIPTION
**Pull request by maxammann**

### CODE REVIEW

File: locales/README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Locales for the Integreat frontend
+
+## Usage
```
Comment (klinzo) [inline locales/README.md:3]: Imo it is not really clear, what has to be done only for the first setup of a repo and what has to be done every time you check out a repo. Maybe you could split this. Also imo nearly everything from `Pushing changes` should be part of `Pulling changes`. The only relevant part there should be `When pushing changes you are required to do the changes directly in the `integreat-locales` repository. After that you can pull the changes.`

File: locales/manage [hunk 2]
```diff
@@ -0,0 +1,188 @@
+#!/usr/bin/node
```
[the remark this answers is not in the document]
Comment (klinzo) [reply locales/manage:1]: I didn't see you've created a separate readme for this, so it should be fine. Feel free to just resolve this comment.

File: CODEOWNERS [hunk 3]
```diff
@@ -4,4 +4,5 @@
 # the repo. Unless a later match takes precedence,
 # these owners will be requested for
 # review when someone opens a pull request.
-*       @maxammann @nklug @klinzo @schedulaar @a-kud @SimonVortkamp @lulo4907
+*       @maxammann @schedulaar @klinzo @nklug @a-kud @Taggotty @lulo4907 @TechPiranja @SimonVortkamp
```
[the remark this answers is not in the document]
Comment (maxammann) [reply CODEOWNERS:7]: why? :D

File: locales/tools/task.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+/*
```
[the remark this answers is not in the document]
Comment (maxammann) [reply locales/tools/task.js:1]: This is used by the find script in the same directory, there is also a comment what it is.

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+* <vortkamp@integreat-app.de>
+* <stricker@integreat-app.de>
+* <seeberg@integreat-app.de>
+
+If you encounter any issues with the app you can also just create a pull request and we'll review
+it and give you feedback!
+
+# Merging
+Before you can merge the code has to be reviewed and the status checks need to pass.
+The CI runs the linter, type-checks and all unit tests. So better run these locally first before pushing.
```
[the remark this answers is not in the document]
Comment (maxammann) [reply .github/CONTRIBUTING.md]: I added a not about creating PRs externally.

File: locales/.gitignore [hunk 6]
```diff
@@ -0,0 +1,104 @@
+# Logs
```
[the remark this answers is not in the document]
Comment (maxammann) [reply locales/.gitignore:1]: This is from a template, so I'd leave it.

File: locales/README.md (hunk 1 above)
Comment (maxammann) [reply locales/README.md:3]: I removed some redundancy at the end of pushing: https://github.com/Integreat/integreat-locales/blob/7ae42356ab25b9a5248bb9e978fa38d9fa258021/README.md#pushing-changes

I want to go over the whole process of pushing changes to the locales. This also includes getting the changes back. Also the pull command is different in both sections as I'm pulling from different branches.

Comment (klinzo) [reply locales/README.md:3]: Why don't you use just a generic branch name, e.g. `<Branch>`. I still think this Readme is to complicated and contains unnecessary duplicates. I'll open a PR with suggestions.

File: locales/README.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+
+As the pull command is quite long you can define an alias:
+```bash
+git config alias.pull-locales "\!f() { git subtree pull --prefix locales locales \$1 --squash -m 'Merging squashed locales for branch \$(git rev-parse --abbrev-ref HEAD)'; }; f"
+```
+
+Then you can pull using:
+
+```bash
+git pull-locales NATIVE-X
```
Comment (klinzo) [inline locales/README.md:78]: I get the following error when trying to use the alias: `expansion of alias 'pull-locales' failed; '!f()' is not a git command`

File: locales/README.md (hunk 1 above)
Comment (klinzo) [reply locales/README.md:3]: PR opened: https://github.com/Integreat/integreat-locales/pull/8

File: locales/README.md (hunk 7 above)
Comment (maxammann) [reply locales/README.md:78]: There was a problem with bash support. I fixed it +1

File: locales/README.md (hunk 1 above)
Comment (maxammann) [reply locales/README.md:3]: Thanks I added your recommendations

File: locales/README.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+**Note that you should not rebase the new branch after you pulled it into another repository.**
+
+After merging `<branch>` into `master` it is possible for anyone to pull again from `master` like described in [Pulling Changes](#pushing-changes). This step is optional for you right now as it will be pulled by the next person which creates a PR.
+
+### Tips & Tricks
+
+As the pull command is quite long you can define an alias:
+```bash
+git config alias.pull-locales '!f() { git subtree pull --prefix locales locales $1 --squash -m "Merging squashed locales for branch $(git rev-parse --abbrev-ref HEAD)"; }; f'
```
Comment (lulo4907) [inline locales/README.md:67]: This command runs into an usage hint about `git config`. It seems to need a location flag.

Comment (maxammann) [reply locales/README.md:67]: Can you paste the usage hint? I do not get a hint.

File: locales/csv/.gitignore [hunk 9]
```diff
@@ -0,0 +1 @@
+*.csv
```
Comment (Taggotty) [inline locales/csv/.gitignore:1]: why does this file exist?

File: locales/README.md (hunk 8 above)
Comment (lulo4907) [reply locales/README.md:67]: I got the bunch of `git config` command option hints. After researching `Git Aliases`, it seems to need a leading flag like `--global` etc. But unfortunately I got the same result with the flag :/

Comment (klinzo) [reply locales/README.md:67]: I can't reproduce, everything fine for me.

File: locales/csv/.gitignore (hunk 9 above)
Comment (maxammann) [reply locales/csv/.gitignore:1]: CSVs generated here should never be commited. They are just intermediate steps for generating locales via the `manage` script.

File: locales/README.md (hunk 8 above)
Comment (maxammann) [reply locales/README.md:67]: Seems like a local environment problem. I will merge this now.

**Pull request merged**